modules/apps/asset/asset-service/src/main/java/com/liferay/asset/internal/util/AssetHelperImpl.java

```diff
@@ -622,11 +631,6 @@ private String _getOrderByCol(
 					sb.append(StringPool.UNDERLINE);
 				}
 			}
-			catch (Exception exception) {
-				_log.error("Unable to sort assets", exception);
-
-				throw exception;
-			}
```

Comment:
@jorgediaz-lr, I added this `try / catch` to get a possible `ArrayIndexOutOfBoundsException` during the execution of line `sortField.split(DDMIndexer.DDM_FIELD_SEPARATOR)[1]`, are you handling this use case in other place of the code? Can you please clarify to me? 🙂